tests/test_close_pr.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+from gidgethub import sansio
+
+from bedevere import close_pr
+
+
+class FakeGH:
+
+    def __init__(self, *, getitem=None):
+        self._getitem_return = getitem
+        self.patch_url = None
+        self.patch_data = None
+        self.data = None
+
+    async def patch(self, url, data):
+        self.patch_url = url
+        self.patch_data = data
+
+
+@pytest.mark.asyncio
+async def test_close_invalid_pr():
+    data = {
+        "pull_request": {
+            "statuses_url": "https://api.github.com/blah/blah/git-sha",
+            "title": "No issue in title",
+            "issue_url": "issue URL",
+            "url": "https://api.github.com/org/repo/pulls/123",
+            "head": {
+                "label": "python:3.6"
+            },
+            "base": {
+                "label": "python:master"
+            }
+        },
+    }
+    pr_data = {
+        "labels": [
+            {"name": "non-trivial"},
+        ]
+    }
+    event = sansio.Event(data, event="pull_request", delivery_id="12345")
+    gh = FakeGH(getitem=pr_data)
+    await close_pr.close_invalid_pr(event, gh)
```

Comment:
To help test the router for the module, it's best to call through it, e.g.:
https://github.com/python/bedevere/blob/48cc64330307bd71fa9573fc7f597058e5b7ad13/tests/test_bpo.py#L63

I've learned this lesson the hard way. 😉